addon/mixins/registration-actions.js

```diff
@@ -0,0 +1,68 @@
+
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+    node: null,
+    draft: null,
+    model: null,
+    _node: Ember.computed.or('node'),
```

Comment:
Using `_node` is probably unneeded if there is a just one attribute for the node.